sys/dev/atkbdc/atkbdc.c

```diff
@@ -57,7 +57,10 @@
 
 /* constants */
 
-#define MAXKBDC		1		/* XXX */
+#define MAXKBDC				1	/* XXX */
+#define MILISECOND_MULTIPLIER		1000	/* micro second values will be multiplied to convert to mili seconds */
```

Comment:
typo, 'milli'. also, be mindful about line lengths, in general, we want to fit into 80 chars